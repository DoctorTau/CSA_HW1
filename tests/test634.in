525
2702 5121 -5806 -911 -1488 3569 -3669 2293 -2921 -8827 5645 -3363 -8167 6198 337 -7607 -4713 4660 -4377 -2202 -5146 6763 -8606 -4836 -9090 -6320 5567 -3164 -8077 -2839 6422 -5042 -8038 -5686 -4677 -7851 5463 9473 8070 7896 7351 -8835 -1925 4249 -6777 3595 -2078 2824 -6375 -7457 -1063 -9294 9848 -762 -5794 6160 2205 3904 4024 5831 -8982 8485 -694 9455 -1343 8522 -3859 -4366 -6308 -4709 -6986 3510 2797 -6405 8172 4900 -5655 9332 -8366 -3726 -8592 4879 -772 -2817 2482 -7115 8777 -9136 -3064 9226 -7079 -9435 -513 -4415 100 -1503 -4237 -7162 8053 1618 6706 3133 8571 5788 2693 755 -8172 -2277 -2678 4001 1979 -1952 -3806 8161 -7805 -5542 -8964 -5135 5814 7823 6220 4963 6539 2577 -3954 -2215 -6863 6247 9892 2630 2888 9381 7434 -991 4732 7464 2130 -4313 -7872 8046 4216 5380 6482 -208 -8811 7078 -9900 -3505 9131 9280 153 5346 7657 6742 234 -4787 -2355 -3505 3520 -7813 7126 -6921 -9660 -6345 -729 -1768 -1095 6122 7986 -2665 4418 9405 -383 -1857 6322 4651 9365 -2228 2573 -1908 5908 -2288 3644 -1102 8186 9279 6914 -3132 -4598 9066 -4752 -9091 7319 -3170 745 -5194 9935 -8094 5938 -5164 9719 6398 -5563 -7275 7289 2546 7494 -3673 5254 -9816 6195 4376 -387 4401 361 9414 8532 4887 5479 4162 9954 3226 6697 2948 -8763 -9446 6379 -2308 -2482 -6940 -5637 -7483 5470 -7557 3442 -2356 -7943 -7162 -96 -3692 -3769 1366 -7111 -6501 5563 -1260 3250 4541 -3947 -1790 -2252 6916 -3277 7409 9904 -4811 750 -8208 -6383 344 5800 6073 5394 -8430 -9222 9272 8869 8144 -9673 8379 -662 874 8395 602 -240 8231 4026 4475 4458 -917 9650 8887 -5320 9705 8631 2446 4658 6799 7301 -3503 1750 4278 7000 -4264 -8261 6440 -5625 4865 -9234 -6428 3473 -2285 6541 -2158 4378 -901 -3717 -4002 -7500 -1136 -2254 5670 -3574 -1397 -6308 8322 5374 -4390 -2992 5136 401 3771 6772 -5172 -3469 -7432 -3737 -2951 -5737 8882 -756 -8616 -831 -612 -9910 -9666 9052 5519 -4338 -466 -4690 4298 9796 9201 2454 -845 445 2702 -4705 4222 -4789 -2685 -3048 2436 -8016 -7385 2148 2405 -4680 2119 5863 4419 -6925 -8142 -4365 -9271 8266 7835 4206 6456 8377 -6021 222 2231 -9140 -9453 1061 2671 9521 8735 9546 2241 -7995 9382 7987 -127 -6278 -3696 2434 8861 -1367 -2106 -3094 6110 -4024 1982 -9142 8250 -9577 -398 6150 8618 1504 5042 6338 -1736 -5517 -3789 -7573 5379 -3175 -7601 -8343 -3692 -9488 -466 -6906 5685 -3761 7427 9911 2881 -4893 -1697 589 -7856 5481 549 2739 7425 6602 5535 -3851 8776 -9197 5537 -9848 6316 -4069 -1403 6332 -5876 2254 -2663 -6209 2865 -3634 -8530 8624 -4966 8477 244 -7937 -3729 7017 3742 -5153 2320 870 -5640 -4883 -2843 7141 9122 1519 -5083 2826 -4005 -4111 4411 9213 -2868 9053 2956 9865 -6409 -2098 7380 -4383 3709 -2353 -7098 -4414 -118 -9882 5786 2466 9223 7617 8450 -4814 -7744 -7449 -9383 6475 -9101 -823 7415 3591 3164 2395 9481 7610 -5496 -1096 2732 4542 7974 -7414 -2557 634 -6658 -4735 -3030 579 3762 3848 -9748 -4462 -3108 -3491 -1763 -3506 -8238 8353 