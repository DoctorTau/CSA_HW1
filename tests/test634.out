-5806 -1488 -2202 -5146 -8606 -4836 -9090 -6320 -3164 -5042 -8038 -5686 -2078 -9294 -762 -5794 -8982 -694 -4366 -6308 -6986 -8366 -3726 -8592 -772 -9136 -3064 -7162 -8172 -2678 -1952 -3806 -5542 -8964 -3954 -7872 -208 -9900 -9660 -1768 -2228 -1908 -2288 -1102 -3132 -4598 -4752 -3170 -5194 -8094 -5164 -9816 -9446 -2308 -2482 -6940 -2356 -7162 -96 -3692 -1260 -1790 -2252 -8208 -8430 -9222 -662 -240 -5320 -4264 -9234 -6428 -2158 -4002 -7500 -1136 -2254 -3574 -6308 -4390 -2992 -5172 -7432 -756 -8616 -612 -9910 -9666 -4338 -466 -4690 -3048 -8016 -4680 -8142 -9140 -6278 -3696 -2106 -3094 -4024 -9142 -398 -1736 -3692 -9488 -466 -6906 -7856 -9848 -5876 -3634 -8530 -4966 -5640 -2868 -2098 -7098 -4414 -118 -9882 -4814 -7744 -5496 -1096 -7414 -6658 -3030 -9748 -4462 -3108 -3506 -8238 